cmd/process-agent/main_common.go

```diff
@@ -140,6 +141,11 @@ func runApp(exit chan struct{}, globalParams *command.GlobalParams) error {
 		}),
 	)
 
+	if err := app.Err(); err != nil {
+		fmt.Println("Failed to initialize the process agent: ", fxutil.UnwrapIfErrArgumentsFailed(err))
```

Comment:
Shouldn't this be printed as a log for better tracking? I imagine that this is something that we'd like to capture in the flare for example.

Reply:
So the reason that I didn't use the logger here is because there is no guarantee that it has successfully initialized at this point. One thing we could do is try to use the logger and fall back to stdout if it's nil. Let me add this functionality